test/integration/kueuectl/list_test.go

```diff
@@ -69,48 +69,50 @@ var _ = ginkgo.Describe("Kueuectl List", ginkgo.Ordered, ginkgo.ContinueOnFailur
 
 		// Simple client set that are using on unit tests not allow to filter by field selector.
 		ginkgo.It("Should print local queues list filtered by field selector", func() {
-			streams, _, output, errOutput := genericiooptions.NewTestIOStreams()
-			configFlags := CreateConfigFlagsWithRestConfig(cfg, streams)
-			executeTime := time.Now()
-			kueuectl := app.NewKueuectlCmd(app.KueuectlOptions{ConfigFlags: configFlags, IOStreams: streams, Clock: testingclock.NewFakeClock(executeTime)})
-
-			kueuectl.SetArgs([]string{"list", "localqueue", "--field-selector",
-				fmt.Sprintf("metadata.name=%s", lq1.Name), "--namespace", ns.Name})
+			gomega.Eventually(func(g gomega.Gomega) {
```

Comment:
Why is Eventually needed since we use a fake clock? Is it that some places in the code still use real clock?

Reply:
Added eventually to fix [this](https://github.com/kubernetes-sigs/kueue/issues/2353#issuecomment-2159897688) one. In another cases you right it's not need. So reverted this changes.

Reply:
Moved this fix to separate PR #2399